resources/templates/emails/pmc-report.ftl

```diff
@@ -96,30 +96,79 @@
+
+  <p>Please note that some metrics required for quarterly board reports must be tracked and calculated by PMCs themselves. For more information see the “Where to Find Program Measures and Data“ section <a href="https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/93225748/Board+Reporting+and+Program+Health+Checks">in the Board Reporting and Program Health Check page in the FINOS Community Handbook</a>.</p>
+
+  <p>In addition to the data available in the <a href="https://metrics.finos.org/app/kibana?#/dashboard/0e542930-4f2d-11e9-9e7c-eb1eab055f1f?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-6m,mode:quick,to:now))&_a=(filters:!(),query:(query_string:(analyze_wildcard:!t,query:'cm_program:%22${program.program_short_name}%22')))">metrics dashboard</a>, please also review both the basic data and program steering warnings and measures below. For more information on how to interpret and follow-up on the steering data, see the <a href="https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/118292491/PMC+Monthly+Reports">PMC Monthly Reports page</a> of the FINOS Community Handbook, which includes recommended remedies and interventions.</p>
+
+  <hr/>
+  <h2>${program.program_name} Program Master Data</h2>
+  <h3>PMC Lead</h3>
+  <ul><li>${pmc_lead}</li></ul>
+  
+  [#if pmc_list?? && pmc_list?size > 0]
+    <h3>PMC Composition</h3>
+    <ul>
+      [#list pmc_list as pmc_person]
+        <li>${pmc_person}</li>
+      [/#list]
+    </ul>
+  [/#if]
+  
+  [#if orgs_in_pmc?? && orgs_in_pmc?size > 0]
+    <h3>Organisations in PMC</h3>
+    <ul>
+      [#list orgs_in_pmc as pmc_org]
+        <li>${pmc_org}</li>
+      [/#list]
+    </ul>
+  [/#if]
+
+  [#if working_groups?? && working_groups?size > 0]
```

Comment:
This should probably go below the heading, and an ```[#else]``` clause added which says something to the effect of "This program currently has no working groups.".